pyproject.toml

```diff
@@ -106,8 +105,7 @@ optional = [
     "h5py>=3.11.0",
     "jarvis-tools>=2020.7.14",
     "matgl>=1.1.1",
-    # TODO: track https://github.com/matplotlib/matplotlib/issues/28551
-    "matplotlib>=3.8,!=3.9.1",
+    "matplotlib>=3.8",
```

Comment:
just to confirm, reading https://github.com/matplotlib/matplotlib/issues/28551 the entire faulty 3.9.1 release was yanked?

Reply:
Yes, see https://pypi.org/project/matplotlib/#history:
<img width="791" alt="image" src="https://github.com/user-attachments/assets/0c435081-0970-4cfc-8ef1-373e513e7297">